assets/airports/ulli.json

```diff
@@ -0,0 +1,2695 @@
+                ["BEPOL*", "LI765", "LI766", "LI761", "LI762", "LI763", "LI764", "ARNAT"]
+            ]
+        },
+        "GENPA1A": {
+            "icao": "GENPA1A",
+            "name": "Genpa One Alpha",
+            "entryPoints": {
+                "GENPA": [["GENPA", "A260-"]]
+            },
+            "body": [
+                ["LI721", "A130+|A260-|S225-"],
+                ["LI722", "A110+|A160-"],
+                ["LI723", "A50+|A80-|S225-"],
+                ["LI724", "A50|S225"],
+                ["LI725", "A50|S225"],
+                ["LI726", "S225"],
+                ["LI727", "S225"],
+                ["LI728", "S225"],
+                ["LI729", "A50|S225"],
+                ["DIKOM", "A25+|A35-"]
+            ],
+            "rwy": {
+                "ULLI10L": [["LEPTO", "A25"]],
+                "ULLI10R": [["GEBKA", "A25"]]
+            },
+            "draw": [
+                ["GENPA*", "LI721", "LI722", "LI723", "LI724", "LI725", "LI726", "LI727", "LI728", "LI729", "DIKOM"]
+            ]
+        },
+        "GENPA1B": {
+            "icao": "GENPA1B",
+            "name": "Genpa One Bravo",
+            "entryPoints": {
+                "GENPA": [["GENPA", "A160-"]]
+            },
+            "body": [
+                ["LI753", "A60+|A160-"],
+                ["LI754", "A50|S225"],
+                ["LI749", "S225"],
+                ["LI750", "S225"],
+                ["LI751", "S225"],
+                ["LI752", "A50|S225"],
+                ["OBARI", "A25+|A35-"]
+            ],
+            "rwy": {
+                "ULLI28L": [["BIPRI", "A25"]],
+                "ULLI28R": [["MADID", "A25"]]
+            },
+            "draw": [
+                ["GENPA*", "LI753", "LI754", "LI749", "LI750", "LI751", "LI752", "OBARI"]
+            ]
+        },
+        "KOLET1A": {
+            "icao": "KOLET1A",
+            "name": "Kolet One Alpha",
+            "entryPoints": {
+                "KOLET": [["KOLET", "A150-"]]
+            },
+            "body": [
+                ["LI730", "A60+|A150-"],
+                ["LI724", "A50|S225"],
+                ["LI725", "A50|S225"],
+                ["LI726", "S225"],
+                ["LI727", "S225"],
+                ["LI728", "S225"],
+                ["LI729", "A50|S225"],
+                ["DIKOM", "A25+|A35-"]
+            ],
+            "rwy": {
+                "ULLI10L": [["LEPTO", "A25"]],
+                "ULLI10R": [["GEBKA", "A25"]]
+            },
+            "draw": [
+                ["KOLET*", "LI730", "LI724", "LI725", "LI726", "LI727", "LI728", "LI729", "DIKOM"]
+            ]
+        },
+        "KOLET1B": {
+            "icao": "KOLET1B",
+            "name": "Kolet One Bravo",
+            "entryPoints": {
+                "KOLET": [["KOLET", "A250-"]]
+            },
+            "body": [
+                ["LI755", "A60+|A250-|S225-"],
+                ["LI746", "A60"],
+                ["LI747", "A50+|A60-|S225-"],
```

Comment:
The *altitude* restrictions at LI747 are for holding only, not part of the KOLET1B arrival